contracts/voting-verifier/src/msg.rs

```diff
@@ -76,7 +76,7 @@ pub enum QueryMsg {
     #[returns(Poll)]
     GetPoll { poll_id: PollId },
 
-    #[returns(Vec<(CrossChainId, VerificationStatus)>)]
+    #[returns(Vec<(Message, VerificationStatus)>)]
```

Comment:
What is the reason for this change again? I know we talked about it in standup but I can't recall

Reply:
besides what we just talked about in the standup today, I see another advantage of having this change. One of the issues I ran into when removing the aggregate-verifier was that I couldn't swap directly the aggregate-verifier client for the one in the voting-verifier because this transformation was done in one but not the other. Changing the return type directly in the API will allow us in the future to easily swap one verifier for another if we want to, or if we return to the aggregate-verifier in the future it would be easier to plug it in if all verifier clients behave the same and no mapping is needed.